frontend/src/app/modules/bookings/bookings-page/bookings-page.component.ts

```diff
@@ -42,11 +64,70 @@ export class BookingsPageComponent extends BaseComponent implements OnInit {
             );
     }
 
-    public goToPage(pageName: string) {
+    @HostListener('window:resize', ['$event'])
+    onResize(event: Event) {
+        const eventWindowTarget = event.target as Window;
+
+        const width = eventWindowTarget.innerWidth;
+
+        this.getNumberOfItemsToDisplayOnResize(width);
+    }
+
+    goToPage(pageName: string) {
         this.router.navigate([`${pageName}`]);
     }
 
     followMeetingLink(link: string) {
         window.open(link);
     }
+
+    private getNumberOfItemsToDisplay() {
+        const containerWidth = this.getPageSize();
+
+        switch (true) {
+            case containerWidth > desktopMaxWidth
```

Comment:
I think you can change switch cases order and start from smaller screens. It will allow you to simplify switch case